.github/workflows/deploy.yml

```
@@ -0,0 +1,58 @@
+name: Deploy to EC2
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - feat-add-deployment-script # to be removed before merging into develop
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)